### ISSUE TITLE
Wrong alt text on image in Disclosure page

### ISSUE THREAD

**Issue by carmacleod** [carbon-design-system/carbon-website#2699]
https://v11.carbondesignsystem.com/patterns/disclosures-pattern/#functionality

The image under "Mouse" needs to have a caption, and its alt text incorrectly says "Example of a simple and complex profile menu.", which is the caption and alt text for the previous image.

**Comment by carmacleod** [comment]
Also, same for the image in the Structure subsection of the Settings and filter menus section.
https://v11.carbondesignsystem.com/patterns/disclosures-pattern/#settings-and-filter-menus

The alt text is "Example of a settings menu structure." but it should be "Example of a filter menu structure." and this image needs a caption.

**Comment by carmacleod** [comment]
Also, same for the image in the Functionality - Mouse subsection of the Settings and filter menus section.
https://v11.carbondesignsystem.com/patterns/disclosures-pattern/#settings-and-filter-menus

The alt text is "Example of a settings menu structure.", which is incorrect, and this image needs a caption.
